### PR TITLE
Edits based on customer feedback

### PR DESCRIPTION
**Pull request by Chuxel**
https://github.com/Microsoft/vscode-remote-release/issues/155
https://github.com/Microsoft/vscode-remote-release/issues/116
https://github.com/Microsoft/vscode-remote-release/issues/95
https://github.com/Microsoft/vscode-remote-release/issues/119

### CODE REVIEW

File: docs/remote/troubleshooting.md [hunk 1]
```diff
@@ -195,14 +195,33 @@ SSH can be strict about file permissions and if they are set incorrectly, you ma
 
 ### Local SSH file and folder permissions
 
+#### macOS / Linux
```
Comment (gregvanl) [inline docs/remote/troubleshooting.md]: H4s don't render very well on our site so I've been changing platform headers to bold text.

**macOS / Linux:**
 
I can update after the merge.

**Comment by Chuxel** [comment]
@gregvanl Sounds good!  Good to know - will do that in future updates.

**Comment by gregvanl** [comment]
Published

**Comment by adlermedrado** [comment]
+1 for microsoft/vscode-remote-release#116

**Pull request merged**